Formula/t/teller.rb

```diff
@@ -1,11 +1,10 @@
 class Teller < Formula
   desc "Secrets management tool for developers built in Go"
   homepage "https://tlr.dev/"
```

Comment:
Do you think we should update the homepage too, maybe use the GitHub repo?

It looks like tlr.dev is broken and they're not planning to revive it: https://github.com/tellerops/teller/issues/251#issuecomment-2102319093

Reply:
yeah, totally.